### PR TITLE
Toolbar Headline

### PR DESCRIPTION
**Pull request by yayaismaail**
In this Project I:

- set up linters
- update README file
- added index.html and styles.css files and 
- built header/toolbar and headline of the portfolio

NOTE: the web hint is not included in the linters because it is not working and we were told to remove the configurations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.css" />
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="#" class="my-logo">MY LOGO</a>
```
Comment (algerina) [inline index.html:14]: - [ Optional] I will recomend you to use your own Logo or create one, it will look more professional.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    
+            <div class="mobile-menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
```
Comment (algerina) [inline index.html:19]: - [ ] The hamburger button should be on the right and has a `nav` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices. I recommend putting your logo outside of the Nav.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+.headline {
+  background: #fff url("images/header-shapes-mobile.png") no-repeat;
+
+  /* background-size: contain; */
+  background-size: 100%;
+  display: flex;
+  flex-direction: column;
+
+  /* height: 100%; */
```
Comment (algerina) [inline styles.css]: - [ ] I recommend deleting commented lines of code, your work is well organized it will look perfect.

File: README.md [hunk 4]
```diff
@@ -1 +1,170 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- - [Live Demo on Netlify](https://) -->
+<!-- - [Live Demo on Github pages](https:///) -->
+
```
Comment (algerina) [inline README.md]: - [ ] Since your already deployed your project, please add the link to the Readme.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @yayaismaail  ,

Great job so far! 

May I highlight the following: 
- [x] Your work is clean and well organized👏  ✔️ 
- [x] Your project looks awesome✔️ 
- [x] Well done deploying already your Portfolio✔️ 




## STATUS: CHANGES REQUIRED:muscle::clap:



 
Check the comments under the review.
 ## Optional suggestions

- [ optional] Your commits are descriptive however it is recommended to use Imperative mood , so instead of "updated Readme" you can use "Update Readme instead".

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 1 above)
Comment (yayaismaail) [reply index.html:14]: @algerina, thank you for your wonderful review, but in the curriculum we were told to replicate the chosen template that is why I used it (MY LOGO) 👍

File: styles.css (hunk 3 above)
Comment (yayaismaail) [reply styles.css]: @algerina, accept my deepest thanks

File: README.md (hunk 4 above)
Comment (yayaismaail) [reply README.md]: @algerina, 😆

Comment (AlphaNtihinduka) [reply README.md]: - [ ] As suggested by the previous reviewer, kindly consider adding the live demo link of the application. You should not leave any section of the readme empty ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @yayaismaail 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your work is clean and well organized👏 ✔️
- Your project looks awesome✔️
- Well done deploying already your Portfolio✔️
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- _Kindly consider the inline review comments ☺️._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 4 above)
Comment (yayaismaail) [reply README.md]: @AlphaNtihinduka, 👍

**Review by divinecharlotte — APPROVED** [review]
# Hello @yayaismaail       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive commit messages👍 
- Good implementation of the required changes 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**